### PR TITLE
Repackage Jackson Serializers/Deserializers and add constants

### PR DESCRIPTION
**Pull request by rafaelraposospot**
# TL;DR

Repackage Classes and add Constants.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by andresgomezfrr — APPROVED** [review]

File: flytekit-jackson/src/main/java/org/flyte/flytekit/jackson/util/JacksonConstants.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.flyte.flytekit.jackson.util;
+
+public class JacksonConstants {
```
Comment (narape) [inline flytekit-jackson/src/main/java/org/flyte/flytekit/jackson/util/JacksonConstants.java:19]:
```suggestion
public class LiteralSerializationProtocol {
```
I think this is a more descriptive name for the purpose of the constants
